app/create_file.py

```diff
@@ -1 +1,41 @@
-# write your code here
+import os
+import sys
+import datetime
+
+
+def create_file(filename: str) -> None:
+    count = 0
+    if os.path.exists(filename):
+        with open(filename, "r+") as file:
+            content = file.read()
+            file.write(content)
+
+    with open(filename, "a") as file:
+        file.write(datetime.datetime.now().
+                   strftime("%Y-%m-%d %H:%M:%S") + "\n")
+        while True:
+            count += 1
+            terminal_line = input("Enter content line:")
```

Comment:
Add a space here to distinguish input
```suggestion
            terminal_line = input("Enter content line: ")
```